pkg/handler/configs.go

```diff
@@ -516,3 +534,20 @@ func (h *Handler) ConfigDefaultToken(c *gin.Context) {
 
 	c.JSON(http.StatusOK, gin.H{"message": "OK"})
 }
+
+func (h *Handler) RemoveDefaultToken(c *gin.Context) {
```

Comment:
body request should be used for POST
in DELETE and GET, i recommend 
- query params `/api/v1/tokens/configs?guild_id=abc`
- path params `/api/v1/tokens/configs/:abc`

Reply:
sorry my bad